code/modules/mob/living/carbon/xenomorph/castes/Lurker.dm

```diff
@@ -137,6 +137,17 @@
 		if(LIA && istype(LIA))
 			LIA.end_cooldown()
 
+/mob/living/carbon/xenomorph/lurker/update_canmove()
```

Comment:
Uh that's a no for me. 

If you really wanted to do this, should be using `COMSIG_MOB_POST_UPDATE_CANMOVE` - and arguably, this is still backwards and incorrect versus the description of your PR which is to disable on stuns. Don't tie this in to movement.

Reply:
For example this would also prevent them from keeping cloak while resting. It also applies to explosions and such. Any sort of immobilization which is not a hard CC will also do it. I strongly suggest not tying this in to movement, because it's not at all what you advertised in the PR.

Reply:
Looking again, a more precise proc to intercept would be /mob/living/proc/handle_stunned() e.g.
`/mob/living/carbon/xenomorph/lurker/handle_stunned()` but yes either registering to COMSIG_MOB_POST_UPDATE_CANMOVE or COMSIG_LIVING_SET_EFFECT are also options (though the former signal IMO is worse than what is already being done here because of signal overhead).